src/rpc/starknet_backend.rs

```diff
@@ -0,0 +1,224 @@
+use crate::rpc::{
+    serializable_types::FeltParam, BlockHashAndNumber, BlockId, BroadcastedDeclareTransaction,
+    BroadcastedDeployAccountTransaction, BroadcastedInvokeTransaction, BroadcastedTransaction,
+    ContractClass, DeclareTransactionResult, DeployAccountTransactionResult, EventFilterWithPage,
+    EventsPage, FeeEstimate, FunctionCall, InvokeTransactionResult, MaybePendingBlockWithTxHashes,
+    MaybePendingBlockWithTxs, MaybePendingTransactionReceipt, StateUpdate,
+    SyncStatusType, Transaction,
+};
+use cairo_felt::Felt252;
+use jsonrpsee::
+    core::{async_trait, RpcResult};
+use starknet_core::types::{TransactionStatus, SimulatedTransaction, SimulationFlag};
+
+use super::StarknetRpcApiServer;
+
+pub struct StarknetBackend {
+    // mempool_handler: Mempool,
+    // store: Store,
+}
```

Comment:
@igaray @ElFantasma this is the struct that would hold the handles to the mempool and the store. All RPC calls get a `&self` to the main `StarknetBackend` object.